### PR TITLE
IMPRO-1563: Updated test_optical_flow

### PR DESCRIPTION
**Pull request by neilCrosswaite**
Udated test_optical_flow to call the test with metadata.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ lib/improver/tests/acceptance/test_nowcast_optical_flow.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4789715&bid=15948224)

File: lib/improver/tests/acceptance/test_nowcast_optical_flow.py [hunk 1]
```diff
@@ -69,10 +69,10 @@ def test_metadata(tmp_path):
     input_paths = [kgo_dir / f"20181103{hhmm}_{RADAR_REGRID}.nc"
                    for hhmm in ("1530", "1545", "1600")]
     oe_path = kgo_dir / OE
-    # TODO: the BATS test does not call improver with the metadata file
-    # metadata_path = kgo_dir / "../metadata/precip.json"
+    metadata_path = kgo_dir / "../metadata/precip.json"
     output_path = tmp_path / "output.nc"
     args = [oe_path, *input_paths,
+            "--attributes-dict", metadata_path,
```
Comment (cgsandford) [inline lib/improver/tests/acceptance/test_nowcast_optical_flow.py]: Looking at the code, optical flow should not do this in the first place (now that it has no "extrapolate" option, and definitely shouldn't use the precipitation nowcast metadata.  Two options here: you could:
- Remove this command line option and update KGO for that, or
- Don't make this change as I will be addressing it anyway in IMPRO-1491

Comment (neilCrosswaite) [reply lib/improver/tests/acceptance/test_nowcast_optical_flow.py]: I will go with the latter.

**Comment by fionaRust** [comment]
Leaving this for @cgsandford to do in IMPRO-1491. Closing PR

**Pull request closed without merging**